### ISSUE TITLE
Code or a browser compatibility issue, this page could not be displayed correctly

### ISSUE THREAD

**Issue by TNTBOMBOM** [mastodon/mastodon#12369]
### Expected behaviour
Login directly without issues

### Actual behaviour
error showing: 
```

Due to a bug in our code or a browser compatibility issue, this page could not be displayed correctly.

Try refreshing the page. If that does not help, you may still be able to use Mastodon through a different browser or native app.

Mastodon v3.0.1
```

- Clipboard showing: 

```
t/<@https://mastodon.technology/packs/js/application-c10d7af9e814e7c452d6.chunk.js:1:31320
pi@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:34:75173
mi@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:34:75715
vi@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:34:77391
zi@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:34:81515
Ki@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:34:85080
Ns/<@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:34:98357
t.unstable_runWithPriority@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:43:3200
Ns@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:34:98291
Fs@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:34:98067
Ps@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:34:97412
As@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:34:98791
Cn@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:34:30606
```
another log: 

```

t/<@https://mastodon.social/packs/js/application-b348a0ea2af8873b3543.chunk.js:1:31320
pi@https://mastodon.social/packs/js/common-2b7ff32cc69e079c896c.js:34:75173
mi@https://mastodon.social/packs/js/common-2b7ff32cc69e079c896c.js:34:75715
vi@https://mastodon.social/packs/js/common-2b7ff32cc69e079c896c.js:34:77391
zi@https://mastodon.social/packs/js/common-2b7ff32cc69e079c896c.js:34:81515
Ki@https://mastodon.social/packs/js/common-2b7ff32cc69e079c896c.js:34:85080
Ns/<@https://mastodon.social/packs/js/common-2b7ff32cc69e079c896c.js:34:98357
t.unstable_runWithPriority@https://mastodon.social/packs/js/common-2b7ff32cc69e079c896c.js:43:3200
Ns@https://mastodon.social/packs/js/common-2b7ff32cc69e079c896c.js:34:98291
Fs@https://mastodon.social/packs/js/common-2b7ff32cc69e079c896c.js:34:98067
Ps@https://mastodon.social/packs/js/common-2b7ff32cc69e079c896c.js:34:97412
Rs@https://mastodon.social/packs/js/common-2b7ff32cc69e079c896c.js:34:98508
notify@https://mastodon.social/packs/js/common-2b7ff32cc69e079c896c.js:1:97185
l</t.notifyNestedSubs@https://mastodon.social/packs/js/common-2b7ff32cc69e079c896c.js:1:96765
l</t.handleChangeWrapper@https://mastodon.social/packs/js/common-2b7ff32cc69e079c896c.js:1:96833
g@https://mastodon.social/packs/js/common-2b7ff32cc69e079c896c.js:13:122214
Ht/</</<@https://mastodon.social/packs/js/common-2b7ff32cc69e079c896c.js:13:108683
Ht/</</<@https://mastodon.social/packs/js/common-2b7ff32cc69e079c896c.js:13:108458
Ht/</</</<@https://mastodon.social/packs/js/common-2b7ff32cc69e079c896c.js:13:108227
a/</</<@https://mastodon.social/packs/js/common-2b7ff32cc69e079c896c.js:13:69256
dispatch@https://mastodon.social/packs/js/common-2b7ff32cc69e079c896c.js:13:124379
j/</<@https://mastodon.social/packs/js/common-2b7ff32cc69e079c896c.js:6:23860
```

another log:

```
t/<@https://mastodon.technology/packs/js/application-c10d7af9e814e7c452d6.chunk.js:1:31320
pi@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:34:75173
mi@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:34:75715
vi@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:34:77391
zi@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:34:81515
Ki@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:34:85080
Ns/<@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:34:98357
t.unstable_runWithPriority@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:43:3200
Ns@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:34:98291
Fs@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:34:98067
Ps@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:34:97412
Rs@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:34:98508
notify@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:1:97189
l</t.notifyNestedSubs@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:1:96769
l</t.handleChangeWrapper@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:1:96837
g@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:13:121110
Ht/</</<@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:13:107579
Ht/</</<@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:13:107354
Ht/</</</<@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:13:107123
a/</</<@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:13:68152
dispatch@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:13:123275
s/</<@https://mastodon.technology/packs/js/common-8bd0b15db6bbb722115a.js:13:124381
```

### Steps to reproduce the problem

use Tor Browser and set security level to safest, try to login to your account.

### Specifications

Mastodon v3.0.1

TorBrowser 9.0.1 , Debian GNU/Linux OS

**Comment by a-dows** [comment]
I’m experiencing the same issue after migrating from mastohost to my own server—issue occurs in all browsers, any network connection that uses my credentials. All my other users can log in fine…

**Comment by STPKITT** [comment]
I have the same issue on Firefox 56.0.2.

**Comment by unixnut** [comment]
Same issue in Firefox 52.0 (actually SeaMonkey/2.49.4) with Mastodon v3.1.1

**Comment by WammKD** [comment]
Same issue on Qutebrowser v1.10.1 with Mastodon v3.1.1; stacktrace being

`TypeError: Object(...)(...).get(...).then(...).catch(...).finally is not a function
    at https://queer.garden/packs/js/common-65a40ba0d18598a0834b.js:2:130805
    at Object.dispatch (https://queer.garden/packs/js/common-65a40ba0d18598a0834b.js:2:285862)
    at t.n.componentWillMount (https://queer.garden/packs/js/application-b2f0b6762b30e044cbc4.chunk.js:1:42132)
    at Io (https://queer.garden/packs/js/common-65a40ba0d18598a0834b.js:2:1011263)
    at Ji (https://queer.garden/packs/js/common-65a40ba0d18598a0834b.js:2:1026983)
    at Ms (https://queer.garden/packs/js/common-65a40ba0d18598a0834b.js:2:1064962)
    at Pu (https://queer.garden/packs/js/common-65a40ba0d18598a0834b.js:2:1049252)
    at Tu (https://queer.garden/packs/js/common-65a40ba0d18598a0834b.js:2:1049177)
    at wu (https://queer.garden/packs/js/common-65a40ba0d18598a0834b.js:2:1046517)
    at gu (https://queer.garden/packs/js/common-65a40ba0d18598a0834b.js:2:1043261)`

**Comment by DaniEra** [comment]
Having the same issue in my instance. Three people who received invites had this bug in the migration. Quoting one of them: 

"Using the Seamonkey, K-Meleon, Pale Moon and Internet Explorer 11 browsers, it is possible to browse the site normally …
EXCEPT WHEN I log in,
because then I get this error message: ‘Due to a bug in our code or a browser compatibility issue, this page could not be displayed correctly.’

This DOESN’T HAPPEN when I am at mastodon.social. Among the browsers available on my PC, the only one ursal.zone does not complain about is Vivaldi, which is a Chromium clone."

**Comment by tsacks** [comment]
Sorry to resurrect an old bug, but I've had this take my server down for the past year. Was able to *just* get it resolved. Issue appears to have cropped up due to a failed asset compile. I'm running a manual install on a vps, so I had to run the following commands should force it to recompile all of the assets:

`su - mastodon
cd ~/live
RAILS_ENV=production bundle exec rake tmp:cache:clear
RAILS_ENV=production bundle exec rake assets:clobber assets:precompile`

Once I did that, and reloaded the page, everything appears to have been running properly. I might have also re-copied over the .service files, and restarted the services, but I don't think I did that between compiling, and things working.
